thermal_processing.py

```diff
@@ -115,14 +116,12 @@ def replace_ext(filename, ext):
     return filename.parent / (filename.stem + ext)
 
 
-def update_metadata(recording, api):
+def update_metadata(conf, recording, api):
     with open(str(recording["filename"]), "rb") as f:
         reader = CPTVReader(f)
         metadata = {}
         metadata["recordingDateTime"] = reader.timestamp.isoformat()
-
-        # TODO Add device name when it can be processed on api server
-        # metadata["device_name"] = reader.device_name
+        metadata["location"] = (reader.latitude, reader.longitude)
```

Comment:
I presume you've double checked that the location ends up in the `location` column in `Recordings`?

Reply:
Some CPTV files won't have lat/lon. You should probably avoid adding `metadata['location']` if they're not set.

Reply:
yep is being set in database, and ye although don't they always have the default lat long of Christchurch?

Reply:
The lat/lon fields are relatively new. Older recordings won't have a CPTV field (when reprocessing). Also, we only assume Christchurch for sunrise/sunset calculations - we won't assume a location otherwise if it hasn't been set.